### PR TITLE
add configuration for weekly ETL

### PR DESCRIPTION
**Pull request by saweber**
## Does this PR rely on any other PRs?

- https://github.com/kubecost/kubecost-cost-model/pull/1315/files
- 

## How does this PR impact users? (This is the kind of thing that goes in release notes!)
see https://github.com/kubecost/kubecost-cost-model/pull/1315

## Links to Issues or ZD tickets this PR addresses or fixes

- https://kubecost.atlassian.net/browse/CORE-109
-

### CODE REVIEW

File: cost-analyzer/values.yaml [hunk 1]
```diff
@@ -375,6 +375,9 @@ kubecostModel:
   # The total number of hours the ETL pipelines will build
   # Set to 0 to disable hourly ETL (not recommended)
   etlHourlyStoreDurationHours: 49
+  # The total number of weeks the ETL pipelines will build
+  # Set to 0 to disable weekly ETL (not recommended)
+  etlWeeklyStoreDurationWeeks: 55
```
Comment (dwbrown2) [inline cost-analyzer/values.yaml]: Nice!! Just for the sake of documenting, why not do 52 or 53 by default?

Comment (saweber) [reply cost-analyzer/values.yaml]: 55 is the stated default in the design document.

Comment (dwbrown2) [reply cost-analyzer/values.yaml]: :) fair, any chance we can add a quick reason why so others can reference in the future? or want to link to the design doc?

Comment (saweber) [reply cost-analyzer/values.yaml]: Great catch! TL;DR - there was a typo that propogated down to me. The original decision from product was precisely 52w (roughly 1 year). We will bump to 53w to ensure 1 full year of coverage by default (52w = 364d, leaving us one day short).

**Review by ghost — APPROVED** [review]
Looks sensible to me

**Review by nikovacevic — APPROVED** [review]

**Pull request merged**